libass/ass_fontselect.c

```diff
@@ -893,6 +935,8 @@ static char *select_font(ASS_FontSelector *priv,
     ASS_FontInfo *matched_font = NULL;
     char *result = NULL;
     bool name_match = false;
+    bool font_queried_via_provider = false;
+    bool font_queried_via_provider_error = false;
```

Comment:
nit: this name is rather long; `font_from_provider` is shorter without loosing information